.gitignore

```diff
@@ -1,4 +1,3 @@
-vendor/*
-/vendor/
-.idea/*
-index.php
+.DS_Store
+.idea
+.vs-code
```

Comment:
@afragen when developing locally, there is always a vendor/ folder hence the need to exclude it from being committed by adding it to .gitignore.

I am in favor of restoring `vendor/*`

Reply:
When developing this library locally, there is always a vendor/ directory. Say you want to make a change, you clone the repo and do `composer install` which creates a vendor directory.

After making the change, if you do `git commit -am "new commit"` the vendor directory will be included. So it's good we exclude it by default.

Make sense?

Reply:
Collins, that's not quite the way I work but if if makes your workflow better. Done.

Reply:
Curious to know how you work?